functions/test/github.test.js

```diff
@@ -1,121 +1,223 @@
+import {
+  addAndGetSnapshot,
+  setAndGetSnapshot,
+  setupDbAdmin,
+  teardownDb
+} from './helpers'
+
+const sha1 = require('sha1')
 const nock = require('nock')
 const fs = require('fs')
 const path = require('path')
 
-const Octokit = require('@octokit/rest')
-
-// Requiring our app implementation
 const Github = require('../lib/github')
+const prOpenedPayload = require('./fixtures/pull_request.opened')
 
-// Requiring our fixtures
-const payload = require('./fixtures/pull_request.opened')
-const hugePayload = require('./fixtures/pull_request.opened.huge')
-const commits = require('./fixtures/commits')
+const contributionKey = 'github.com/bocon13/cla-test/pull/3'
+const contributionId = sha1(contributionKey)
+const identity = 'github:bocon13'
 
-const checkIdentifiesSuccess = async () => {
-  return Promise.resolve({
-    allWhitelisted: true,
-    missingIdentities: []
-  })
-}
+const githubApi = 'https://api.github.com'
+const statusUri = '/repos/bocon13/cla-test/statuses/2129e453f6d652badfb353c510a3669873a15f7c'
+const createCommentUri = '/repos/bocon13/cla-test/issues/3/comments'
+const existingCommentUri = '/repos/bocon13/cla-test/issues/comments/1'
 
-const checkIdentifiesFail = async () => {
-  return Promise.resolve({
-    allWhitelisted: false,
-    missingIdentities: []
-  })
-}
+const appId = 123
+const projectId = 'github-test-' + new Date()
 
 nock.disableNetConnect()
 
-describe('Github PR Webhook', () => {
-  const appId = 123
+describe('Github lib', () => {
+  let db
   let github
+  let contribsRef
+  let eventsRef
+  let whitelistsRef
   let mockCert
-  const cla = {
-    checkIdentities: checkIdentifiesSuccess
-  }
+  let mockContribution
+  let mockEvent
+  let payload
 
-  beforeAll((done) => {
+  beforeAll(async (done) => {
```

Comment:
don't think you need the `async` keyword here

Reply:
Done

Reply:
Actually, I put it back. I think it is needed to avoid this error:
https://app.circleci.com/jobs/github/OpenNetworkingFoundation/cla-manager/208